src/components/BToggle/BToggle.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+
+  <div class="toggle-container">
+    <label class="toggle">
+      <input 
+        :disabled="disabled" 
+        :checked="checked"
+        :aria-pressed="checked"
+        type="checkbox" 
+      >
+      <span class="slider">
+        <label :class="labelClass">{{this.label}}</label>
+      </span>
+    </label>
+  </div>
+  
+</template>
+
+<script>
+export default {
+  name: "b-toggle",
+  props: {
+    id:{
+      type: String,
+      required: true
+    },
+    label:{
+      type: String,
+      required: true
+    },
+    required:{
+      type: Boolean,
+      required: false
+    },
+    disabled:{
+      type: Boolean,
+      required: false
+    },
+    checked:{
+      type: Boolean,
+      required: false
+    }
+  },
+  data () {
+    return {}
+  },
+  computed: {
+    labelClass: function () {
+      return {
+        "disabled": this.disabled,
+        "text": true
+      }
+    }
+  }
+}
+</script>
+
+<style scoped>
+
+  .toggle-container {
+  }  
+
+  .toggle {
+    border-style: solid;
+    border-width: thin;
+    border-radius: .25rem;
+    border-color: #dededf;
+    position: relative;
+    display: inline-block;
+    cursor: pointer;
+    padding-left: 2rem;
+    height: 2.5rem;
+    line-height: 2.5rem;
+    margin: .55rem;
+  }
+
+  .toggle input {
+    display:none;
```

Comment:
Using `display: none` makes the input invisible to screen readers. This needs to do more intentional styling removal instead.